schema/m3-requirements.schema.json

```diff
@@ -392,7 +392,7 @@
                   "description": "The type of resource that will be reduced."
                 },
                 "count": {
-                  "$id": "#/definitions/logicalRequirement/items/properties/resourceAvailable/properties/items/count",
+                  "$id": "#/definitions/logicalRequirement/items/properties/resourceAtMost/properties/items/count",
```

Comment:
Pretty sure this was a copy/paste error, but maybe there is just something I'm misunderstanding.

Reply:
Yeah I think `$id` only needs to be set at the top level (where I notice we're using `id`, which apparently is the old Draft 4 way of doing it, instead of `$id`). The other ones could probably all be removed. It could be done with a script. May be worth discussing on #data-dev?